test/terraform_aws_platform_test.go

```diff
@@ -0,0 +1,61 @@
+package test
+
+import (
+	"os"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTerraformAWSPlatformUsWest2(t *testing.T) {
+	// Construct the terraform options with default retryable errors to handle the most common
+	// retryable errors in terraform testing.
+	os.Setenv("AWS_REGION", "us-west-2")
```

Comment:
This actually tests the same part of the regex as `us-east-1`, I was thinking we might want to add perhaps the following:
- af-south-1
- ap-southeast-2
- ca-central-1
- eu-north-1

Reply:
added!